#include <iostream>
using namespace std;
int main()
{
	int a,b;
  	cin>>a>>b;
  	try
    {
      	if(b==0)
          throw "EXCEPTION Not possible";
        else
          throw "VALID";
    }
  	catch(const char * msg)
    {
      	cout<<msg;
    }
	return 0;
}